src/Stripe.net/Entities/PaymentIntents/PaymentIntentTransferData.cs

```diff
@@ -10,7 +10,7 @@ public class PaymentIntentTransferData : StripeEntity<PaymentIntentTransferData>
         /// charge amount is transferred to the destination account.
         /// </summary>
         [JsonProperty("amount")]
-        public long? Amount { get; set; }
+        public long Amount { get; set; }
```

Comment:
I seem to remember a change like this one breaking users a while ago. I'm sure if the spec says it's not nullable we're good, but I am quite surprised. cc @ob-stripe